### PR TITLE
Update userguide for addProduct, viewClient and viewMeeting

### PR DESCRIPTION
**Pull request by ThomasHoooo**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -277,11 +277,21 @@ View details associated with a client, such as the client's name and phone numbe
 Format: `viewClient i/INDEX`
 
 * Display information about the client at the specific `INDEX`.
-* The index refers to the index number shown in the displayed clients' list.
+* The index refers to the index number shown by executing [`listClient`](#512-list-clients-listclient) command.
 * The index **must be a positive integer** 1, 2, 3, …​
 
+Use case:
+
+1. View more specific details related to the client such as address and birthday which are not shown in clients panel. This will help you foster a stronger relationship with your clients because you can wish them happy birthday!
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:285]: list of clients

File: docs/UserGuide.md [hunk 2]
```diff
@@ -277,11 +277,21 @@ View details associated with a client, such as the client's name and phone numbe
...
+
 Examples:
-* `viewClient i/1`
+
+* View client at index 1
+  * `viewClient i/1`
+
+<div markdown="block" class="alert alert-warning">:exclamation: **Caution:**
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:292]: delete

File: docs/UserGuide.md [hunk 3]
```diff
@@ -277,11 +277,21 @@ View details associated with a client, such as the client's name and phone numbe
 Format: `viewClient i/INDEX`
 
 * Display information about the client at the specific `INDEX`.
-* The index refers to the index number shown in the displayed clients' list.
+* The index refers to the index number shown by executing [`listClient`](#512-list-clients-listclient) command.
 * The index **must be a positive integer** 1, 2, 3, …​
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:281]: `INDEX`

File: docs/UserGuide.md [hunk 4]
```diff
@@ -374,17 +384,25 @@ Examples:
...
 * The index **must be a positive integer** 1, 2, 3, …
 
+Use Case:
+
+1. Get an overview of the meeting and the client you are meeting with. This helps you become more prepared for the meeting and you will not have to remember every single small detail of the meeting!
+
 Examples:
 * `viewMeeting i/1`
 
+<div markdown="block" class="alert alert-warning">:exclamation: **Caution:**
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:402]: delete

File: docs/UserGuide.md [hunk 5]
```diff
@@ -374,17 +384,25 @@ Examples:
...
 
-View details associated with a meeting, such as meeting's date and time.
+View details associated with a meeting, such as the meeting's date and time.
 
 Format: `viewMeeting i/INDEX`
 
-* Display information about the meeting at the specific `INDEX`.
+* Display information about the meeting at the specific `INDEX` and details of the client whom you are meeting.
 * The index refers to the index number shown by executing [`listMeeting`](#522-list-meetings-listmeeting) command.
 * The index **must be a positive integer** 1, 2, 3, …
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:393]: `INDEX`

**Review by sikai00 — APPROVED** [review]

**Pull request merged**